### ISSUE TITLE
Increase contrast of dark anchor links on dark background

### ISSUE THREAD

**Issue by twinkarma** [RSE-Sheffield/RSE-Sheffield.github.io#153]
I've been meaning to do this for a while. I think it's failing an accessibility test according to online checkers e.g. webaim.org

**Comment by twinkarma** [comment]
Done in latest commit
